mender-update/daemon/states.hpp

```diff
@@ -289,7 +289,8 @@ class StateScriptState : virtual public StateType {
 			rootfs_script_path,
 		},
 		state_ {state},
-		action_ {action} {};
+		action_ {action},
+		on_error_ {on_error_} {};
```

Comment:
Nit: the commit message says `ignore_error` but the fix is about `on_error` parameter (which can happen to have an "ignore" value).